samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/print/PDFAssessmentBean.java

```diff
@@ -373,7 +373,8 @@ public String prepDocumentPDF() {
 
 				StringBuffer contentBuffer = new StringBuffer(); 
 
-				if (!(item.getItemData().getTypeId().equals(TypeIfc.FILL_IN_BLANK) || item.getItemData().getTypeId().equals(TypeIfc.FILL_IN_NUMERIC))) {
+				if (!(item.getItemData().getTypeId().equals(TypeIfc.FILL_IN_BLANK) || item.getItemData().getTypeId().equals(TypeIfc.FILL_IN_NUMERIC)  
```

Comment:
Since you're changing these lines, I would make them null safe while you're at it, like with the one below. Ex:

if ( !(TypeIfc.FILL_IN_BLANK.equals(item.getITemData().getTypeId()) || etc etc